src/core/js/views/drawerView.js

```diff
@@ -200,23 +190,16 @@ class DrawerView extends Backbone.View {
   renderItems() {
     Adapt.trigger('drawer:empty');
     this.emptyDrawer();
-    const models = this.collection.models;
-    for (let i = 0, len = models.length; i < len; i++) {
-      const item = models[i];
-      new DrawerItemView({ model: item });
-    }
+    this.collection.models.forEach(model => new DrawerItemView({ model }));
```

Comment:
Unnecessary nested `models` access:
```suggestion
    this.collection.forEach(model => new DrawerItemView({ model }));
```